### PR TITLE
Mark 4.3.0 ibm-vpc-block-driver template unsupported

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

**Review by mssachan — CHANGES_REQUESTED** [review]
Mark version `4.3` as `unsupported` in changelog.json as well
https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json#L48-L51

File: config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json [hunk 1]
```diff
@@ -24,26 +24,7 @@
...
-            "previous-revision": {
-                "revision": "11",
-                "release-date": "17 November 2022",
-                "cves": "CVE-2022-1304,CVE-2016-3709",
-                "golang": "1.18.6",
-                "ubi": "8.7-923",
-                "new-features": [],
-                "fixes": []
-            }
-        }
```
Comment (mssachan) [inline config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json:46]: I think u removed one required `}`

Comment (prankulmahajan) [reply config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json:46]: Updated

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**